tests/kernel/io_uring.pm

```diff
@@ -0,0 +1,140 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Executes liburing testing suite
+# Maintainer: Kernel QE <kernel-qa@suse.de>
+# More documentation is at the bottom
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils;
+
+sub run {
+    my $self = shift;
+
+    select_serial_terminal;
+
+    my $repository = get_var('LIBURING_REPO', 'https://github.com/axboe/liburing.git');
+    my $timeout = get_var('LIBURING_TIMEOUT', 1800);
+    my $version = get_var('LIBURING_VERSION', '');
+    my $exclude = get_var('LIBURING_EXCLUDE', '');
+    my $issues = get_var('LIBURING_KNOWNISSUES', '');
+    my $pkgs = "git-core";
+    my @lines;
+    my $out;
+
+    # check if liburing2 is installed and eventually install it
+    $out = script_output("rpm -q liburing2");
+    if ($out =~ /not installed/) {
+        $pkgs .= " liburing2";
+    }
+
+    # install dependences
+    zypper_call("in -t pattern devel_basis");
+    zypper_call("in $pkgs");
+
+    # select latest liburing version which is supported by the system
+    if ($version eq '') {
+        $out = script_output("rpm -qi liburing2 | awk '/Version/ {print \$3}'");
```

Comment:
Even better: `script_output('rpm -q --qf "${version}\n" liburing2');`

Reply:
Why `$`? I'd expect `rpm -q --qf "%{Version}" liburing2` (even new line is not needed.

Reply:
Yes, sorry, it should be `%{version}`. Fixed. However, if you don't include the newline or some other whitespace and you have multiple package versions installed, all versions will be printed on the same line with no separator. E.g.

    1.11.21.3

instead of

    1.1
    1.2
    1.3

Reply:
Make sense, but int that case it might make sense to sort and print the only one:
```
rpm -q --qf "%{Version}\n" liburing2 | sort -nr | head -1
```
(although having more versions it'd be better to determine the default one and test against it)